### PR TITLE
Fix bug where initializing the code mirror too early caused rendering issues for short files.

### PR DESCRIPTION
**Pull request by jacob314**
Fixes https://github.com/flutter/devtools/issues/808

### CODE REVIEW

File: packages/devtools/lib/src/debugger/debugger.dart [hunk 1]
```diff
@@ -452,6 +430,28 @@ class DebuggerScreen extends Screen {
...
       minSize: [200, 60],
     );
+
+    final options = <String, dynamic>{
+      'mode': 'dart',
+      'lineNumbers': true,
+      'gutters': <String>['breakpoints'],
+    };
+    final CodeMirror codeMirror =
+    CodeMirror.fromElement(_sourceArea.element, options: options);
```
Comment (kenzieschmoll) [inline packages/devtools/lib/src/debugger/debugger.dart]: nit: dartfmt

Comment (jacob314) [reply packages/devtools/lib/src/debugger/debugger.dart]: I wish copy and paste in intellij didn't mess up indentation. :(

File: packages/devtools/lib/src/debugger/debugger.dart [hunk 2]
```diff
@@ -452,6 +430,28 @@ class DebuggerScreen extends Screen {
       sizes: [80, 20],
       minSize: [200, 60],
     );
+
+    final options = <String, dynamic>{
+      'mode': 'dart',
+      'lineNumbers': true,
+      'gutters': <String>['breakpoints'],
+    };
+    final CodeMirror codeMirror =
```
Comment (natebosch) [inline packages/devtools/lib/src/debugger/debugger.dart]: [nit] Remove the type on the left while you're here?

Comment (jacob314) [reply packages/devtools/lib/src/debugger/debugger.dart]: done.

**Pull request merged**